src/test/java/tests/FilterEvalTests.java

```diff
@@ -726,4 +755,24 @@ public void filterLabelMatching() {
         assertFalse(Qualifier.labelMatches("pi.hi", "p.high"));
         assertFalse(Qualifier.labelMatches(".", "p.high"));
     }
+
+    /**
+     * Helper method for testing an issue against a filter string in the context
+     * of an empty model.
+     */
```

Comment:
Follow the same style when commenting. e.g. ~~`Helper method for testing `~~ `Confirms that ...`